src/steps/iNavigateTo.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * This step will set a new hash to the app.
+ * The app should react to the navigation by providing proper routing.
+ *
+ * ToDo: Add [with <JSON>] to synopsis to pass data as json
+ *    regex: "(\\s+with\\s+(.+?))?"
+ *    parsing: var oObject = JSON.parse(sObject);
+ */
+module.exports = {
+    docs: {
+        description: "Performs navigation using hash.",
+        synopsis: "I navigate to <hash>",
+        examples: [
+            "I navigate to /Customers/4711",
+            "I navigate to /Customers/4711?&tab=person"
```

Comment:
- Consider adding more examples, especially with multiple parameters. You can test the examples without additional code via `npm test`, and see if they match the regex.
- In ...4711?&tab=person, was the "&" on purpose? I thought example should come without & although there are URLs with ?& around...
- Also perhaps we should add more example with multiple parameters